docs/helm/examples/external-hazelcast.md

```diff
@@ -0,0 +1,190 @@
+# Alfresco Content Services Helm Deployment with external Hazelcast cluster
+
+- [Alfresco Content Services Helm Deployment with external Hazelcast
+  cluster](#alfresco-content-services-helm-deployment-with-external-hazelcast-cluster).
+  - [Prerequisites](#prerequisites)
+  - [Warning](#warning)
+  - [Step by step guide](#step-by-step-guide)
+  - [Tests](#tests)
+
+## Prerequisites
+
+- Having installed either of:
+  - [Rancher for Desktop](https://rancherdesktop.io/). Includes kubectl and
+    Helm, ready to use right after installation.
+  - [Docker for Desktop](https://docs.docker.com/desktop/). Requires separate
+    install of kubectl and Helm.
+
+## Warning
+
+  This feautre won't be showing proper server details on Alfresco UI in
+  Repository Server Clustering under section Cluster Members. Alfresco
+  repository pod will be still present after scaling it down. Pod with new name
+  will be added to list but the older will still remain. To verify that the
+  solution is working you can check managment center or do manual test described
+  in [Tests](#tests)
+
+## Step by step guide
+
+1. Deploy alfresco-content-services. See [desktop
+   deployment](../desktop-deployment.md) section. Remember to stick to one
+   namespace in next steps.
+
+2. Prepare a valid Hazelcast xml configuration. First step in this [Alfresco
+   docs](https://docs.alfresco.com/content-services/latest/admin/cluster/#set-up-repository-clustering-via-external-hazelcast).
+   If you want to use config generated by us skip to 7th step.
+    - Example file:
+      [caches.properties](https://github.com/Alfresco/alfresco-community-repo/blob/master/repository/src/main/resources/alfresco/caches.properties)
+
+3. Replace the `<cluster-name>` within the `alfresco-hazelcast-config.xml`file
+   with a secure value that is hard to guess. For all intents and purposes this
+   field  should be treated as a password, as if matching it allows
+   client-server / member-member connection.
+
+4. Add appropriate configation for kubernetes autodiscovery feature and [rest
+   api
+   groups](https://docs.hazelcast.com/hazelcast/5.3/maintain-cluster/rest-api#using-rest-endpoint-groups)
+   depending on your choise.
+
+    ```xml
+    <network>
+        <join>
+            <kubernetes enabled="true">
+                <service-name>hazelcast</service-name>
+            </kubernetes>
+        </join>
+        <rest-api enabled="true">
+            <endpoint-group name="CLUSTER_READ" enabled="true"/>
+            <endpoint-group name="CLUSTER_WRITE" enabled="true"/>
+            <endpoint-group name="HEALTH_CHECK" enabled="true"/>
+            <endpoint-group name="HOT_RESTART" enabled="true"/>
+            <endpoint-group name="WAN" enabled="true"/>
+            <endpoint-group name="DATA" enabled="true"/>
+        </rest-api>
```

Comment:
didn't check, but those are necessary for management-center?

Reply:
DATA one is necessary for manual testing for sure but the rest I will check if they influence mancenter.

Reply:
I left only DATA and mancenter is looking all right I guess we can keep DATA and delete rest.